lib/components/Loader/Loader.tsx

```diff
@@ -47,7 +35,11 @@ export const Loader = ({ size = 'standard' }: LoaderProps) => {
         <Box
           key={index}
           borderRadius="full"
-          background={indicatorColour(backgroundContext!)}
+          background={
+            backgroundContext
+              ? resolveBgForContext[backgroundContext]
```

Comment:
Should this fall back to `neutral` if no match is found?

Reply:
Damn, i fixed that but it wasnt staged 🤦‍♂